packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_utils.ts

```diff
@@ -31,6 +35,16 @@ export const getEditingTexts = (options) => {
   };
 };
 
+export const generateNewRowTempKey = function (): string {
+  return NEW_ROW_TEMP_KEY_PREFIX + String(new Guid());
+};
+export const isNewRowTempKey = function (key: string): boolean {
+  return !!key
+    && typeof key === 'string'
+    && key.startsWith(NEW_ROW_TEMP_KEY_PREFIX)
+    && key.length === NEW_ROW_TEMP_KEY_PREFIX.length + GUID_LENGTH;
+};
```

Comment:
Let's use arrow functions here and add the space between them.
And we can use string templates instead of string concatenation:

```suggestion
export const generateNewRowTempKey = () => 
  `${NEW_ROW_TEMP_KEY_PREFIX}${new Guid()}`;
  
export const isNewRowTempKey = (key: string) => 
  !!key
    && typeof key === 'string'
    && key.startsWith(NEW_ROW_TEMP_KEY_PREFIX)
    && key.length === NEW_ROW_TEMP_KEY_PREFIX.length + GUID_LENGTH;
```